cfgov/unprocessed/apps/youth-employment-success/js/index.js

```diff
@@ -24,7 +23,7 @@ const routeOptionForms = expandables.map( ( expandable, index ) => {
 } );
 
 /* only initialize the first form, the other gets initialized when
-   the user clicks 'add another option' button */
+  the user clicks 'add another option' button */
```

Comment:
Did the autofix on the linter do this? Seemed better aligned, but fine if the linter did it.

Reply:
Yeah the linter would have done this, I didn't touch the documentation layout